app/models/user.rb

```diff
@@ -318,6 +318,15 @@ def set_mhv_ids(mhv_id)
 
   # Other MPI
 
+  def validate_mpi_profile
+    errors = []
+    return unless mpi_profile
+
+    errors.push(:mpi_death_flag) unless mpi_profile.deceased_date.nil?
```

Comment:
you can just do `if` for both of these and you won't need `== false` for `id_theft_flag` and you won't need `.nil?` for `deceased_date`, just makes it a bit simpler